fastlane/lib/fastlane/actions/onesignal.rb

```diff
@@ -92,6 +94,12 @@ def self.available_options
                                        sensitive: true,
                                        optional: true),
 
+          FastlaneCore::ConfigItem.new(key: :android_sender_id,
+                                       env_name: "ANDROID_SENDER_ID",
+                                       description: "ANDROID SENDER ID",
```

Comment:
Can you change the config to...
```rb
key: :android_gcm_sender_id,
env_name: "ANDROID_GCM_SENDER_ID",
description: "GCM Sender ID"
```
And then change the example below and code using this up above 🙃 

Just want to make sure its "GCM" since "FCM" is also a thing and who knows if OneSignal will add that some day 😇